docs/guides/config.json_directives.md

```diff
@@ -0,0 +1,143 @@
+
+  Screenshots of logging.use_color set as true and set as false.
+
+  True:
+
+  ![Logging use_colors screenshot true](screenshots/config.json_directives/logging_use_color_true.png "Logging.use_colors true")
+
+  False:
+
+  ![Logging use_colors screenshot false](screenshots/config.json_directives/logging_use_color_false.png "Logging.use_colors false")
+
+#### logging.http.enable
+
+  Enables the capability to log all the information of the HTTP(s) requests made by Concord.
+
+#### logging.http.filename
+
+  The filename of the file that will be used to save the information of the HTTP(s) requests.
+
+  See `logging.http.enable` for more information.
+
+  This will be obsolete if `logging.http.enable` is set to false.
+
+#### logging.disable_modules
+  
+  Disables one of Concord's default logging instead of disabling all of them like logging.quiet.
+
+  Screenshots of disabling "WEBSOCKETS" and also not disabling any logging:
+
+  Disabling "WEBSOCKETS":
+
+  ![Logging disable_modules screenshot websockets](screenshots/config.json_directives/logging_disable_modules_websockets.png "Logging.disable_modules websockets")
+
+  Not disabling any:
+
+  ![Logging disable_modules screenshot none](screenshots/config.json_directives/logging_disable_modules_none.png "Logging.disable_modules none")
+
+### Discord
+
+#### discord.token
+
+  The token that Concord will use to log in to the bot, in case you use the `discord_config_init` to define the `struct discord *client`.
+
+#### discord.default_prefix.enable
+
+  Will enable the opportunity to set a default prefix.
+
+  See `discord.default_prefix.prefix` for more information.
+
+#### discord.default_prefix.prefix
+
+  Set a default prefix, so `discord_set_on_command` function can be used.
+
+  This will not work in case `discord.default_prefix.enable` is set to false.
+
+## Observations
+
+  You can also put custom fields on your config.json and get its value with the `discord_config_get_field` function. See the following example.
+
+  ```c
+  char foo[16];
+
+  struct ccord_szbuf_readonly value = discord_config_get_field(client, (char *[2]){ "field", "foo" }, 2);
+
+  snprintf(foo, sizeof(foo), "%.*s", (int)value.size, value.start);
+  ```
+
+  ```c
+  struct ccord_szbuf_readonly discord_config_get_field(struct discord *client, char *const path[], unsigned depth)
+  ```
+
+  Attention: this function will only work if you initialize your bot with the `discord_config_init`.
+
+  The first argument of `discord_config_get_field` is the `struct discord *client` that `discord_init`/`discord_config_init` (or also, in case the code is inside some kind of event, it might be on the event parameters) returns to you.
+
+  The second argument is a `const x[]` (also known as `array`) the path to the field where you want to get its value.
+
+  The third argument is an `int` value of the depth of the path, for example, if the path was `field -> another_field -> foo` it would be 3
```

Comment:
```suggestion
  The third argument is an `int` value of the depth of the path, for example, if the path was `field -> another_field -> foo` it would be 3
  
*Written by [ThePedro](https://github.com/ThePedroo)*
```